webapi/src/main/scala/org/knora/webapi/core/LayersLive.scala

```diff
@@ -105,7 +106,9 @@ object LayersLive {
       ConstructTransformer.layer,
       DspIngestClientLive.layer,
       GravsearchTypeInspectionRunner.layer,
-      GroupsResponderADMLive.layer,
+      GroupsResponderADM.layer,
+      GroupsRestService.layer,
+      GroupService.layer,
```

Comment:
**issue**: These `Group*Service` layer invocations belong in the respective `Modules`

Reply:
Fixed in [9445614](https://github.com/dasch-swiss/dsp-api/pull/3139/commits/9445614b984938a919c4ba986194c45bc906537e). I was following the Users implementation, which is kind of tided to Groups. Why then `UserService` is not presented in `AdminModule` but is in both test and live layers?